toolbox/incari/object/get-rotation.md

```diff
@@ -0,0 +1,32 @@
+# Overview
+
+![The Get Rotation Node.](../../../.gitbook/assets/toolbox/incari/object/get-rotation.PNG)
+
+| Input | Output |
+| :--- | :--- |
+| Object ID | Rotation |
+
+**Get Rotation** returns the *rotation* of an **Object** in a **Scene**. The **Object ID** is given as input to the **Node**.
+
```

Comment:
# Attributes

|Attribute|Type|Description|
|---|---|---|
|`Object`|**ObjectID**|The **Object** that you wish to return the `rotation` of, if one isn't provided in the  `Object ID` **Socket**.|